inc/database.class.php

```diff
@@ -0,0 +1,320 @@
+      echo "<td>";
+      Group::dropdown([
+         'name'      => 'groups_id_tech',
+         'value'     => $this->fields['groups_id_tech'],
+         'entity'    => $this->fields['entities_id'],
+         'condition' => ['is_assign' => 1],
+         'rand' => $rand
+      ]);
+
+      echo "</td>";
+
+      $rowspan        = 3;
+
+      echo "<td rowspan='$rowspan'><label for='comment'>".__('Comments')."</label></td>";
+      echo "<td rowspan='$rowspan' class='middle'>";
+
+      echo "<textarea cols='45' rows='".($rowspan+2)."' id='comment' name='comment' >".
+         $this->fields["comment"];
+      echo "</textarea></td></tr>";
+
+      echo "<tr class='tab_bg_1'>";
+      echo "<td><label for='dropdown_users_id_tech$rand'>".__('Technician in charge of the hardware')."</label></td>";
+      echo "<td>";
+      User::dropdown(['name'   => 'users_id_tech',
+         'value'  => $this->fields["users_id_tech"],
+         'right'  => 'own_ticket',
+         'entity' => $this->fields["entities_id"],
+         'rand'   => $rand]);
+      echo "</td></tr>";
+      echo "<tr><td><label for='dropdown_manufacturers_id$rand'>".Manufacturer::getTypeName(1)."</label></td>";
+      echo "<td>";
+      Manufacturer::dropdown(['value' => $this->fields["manufacturers_id"], 'rand' => $rand]);
+      echo "</td></tr>\n";
+
+      if ($this->isNewItem($ID)) {
+         echo "<tr class='tab_bg_1'>";
+         echo "<td colspan='4' class='center'>".__('Default instance')."</td>";
+         echo "</tr>";
+         echo "<tr class='tab_bg_1'>";
+         echo "<td><label for='instance_port$rand'>"._n('Port', 'Ports', 1)."</label></td>";
+         echo "<td>";
+         echo Html::input(
+            'instance_port', [
+               'type' => 'number',
+               'id' => 'instance_port'.$rand
+            ]
+         );
+
+         echo "</td>";
+         echo "<td><label for='dropdown_is_onbackup$rand'>".__('Has backup')."</label></td>";
+         echo "<td>";
+         Dropdown::showYesNo('is_onbackup', 0, -1, ['rand' => $rand]);
+         echo "</td></tr>\n";
+      }
+
+      $this->showInventoryInfo();
+
+      $this->showFormButtons($options);
+
+      return true;
+   }
+
+   function rawSearchOptions() {
+
+      $tab = parent::rawSearchOptions();
+
+      $tab = array_merge($tab, Location::rawSearchOptionsToAdd());
+
+      return $tab;
+   }
+
+
+   function post_addItem() {
+      //create default instance
+      $this->getFromDB($this->fields['id']);
+      $instance = new DatabaseInstance();
+      $instance->add([
+         'name' => ($this->input['instance_name'] ?? $this->input['name']),
+         'port' => ($this->input['port'] ?? 0),
+         'size' => ($this->input['size'] ?? 0),
+         'is_onbackup' => ($this->input['is_onbackup'] ?? 0),
+         'is_active' => ($this->input['instance_is_active'] ?? $this->fields['is_active'] ?? 0),
```

Comment:
I guess `$this->fields['is_active']` is always defined.
```suggestion
         'is_active' => ($this->input['instance_is_active'] ?? $this->fields['is_active']),
```